src/Our.Umbraco.Ditto/ComponentModel/TypeConverters/DittoBaseConverter.cs

```diff
@@ -0,0 +1,175 @@
+﻿namespace Our.Umbraco.Ditto
+{
+    using System;
+    using System.Collections.Generic;
+    using System.ComponentModel;
+    using System.Globalization;
+    using System.IO;
+    using System.Xml;
+
+    using global::Umbraco.Core;
+    using global::Umbraco.Core.Models;
+    using global::Umbraco.Web;
+
+    /// <summary>
+    /// The Ditto base converter containing reusable properties and converting methods for <see cref="IPublishedContent"/> instances.
+    /// All other Ditto converters should inherit from this class.
+    /// </summary>
+    public abstract class DittoBaseConverter : TypeConverter
+    {
+        /// <summary>
+        /// Gets the <see cref="UmbracoHelper"/> for querying published content or media.
+        /// </summary>
+        public static UmbracoHelper UmbracoHelper
```

Comment:
Quick follow up, I was looking at how Umbraco core do it for their base/abstract classes...

* [UmbracoController.cs#L30](https://github.com/umbraco/Umbraco-CMS/blob/dev-v7/src/Umbraco.Web/Mvc/UmbracoController.cs#L30)
* [UmbracoControl.cs#L40](https://github.com/umbraco/Umbraco-CMS/blob/dev-v7/src/Umbraco.Web/UI/Controls/UmbracoControl.cs#L40)
* _(there are a few others)_